arcade/arcade/cli/main.py

```diff
@@ -702,6 +703,59 @@ def dashboard(
         )
 
 
+@cli.command(help="Generate Toolkit documentation", rich_help_panel="Tool Development")
+def generate_toolkit_docs(
+    toolkit_name: str = typer.Option(
+        None, "--toolkit-name", "-n", help="The name of the toolkit to generate documentation for."
+    ),
+    docs_section: str = typer.Option(
+        "",
+        "--docs-section",
+        "-s",
+        help=(
+            "The section of the docs to generate documentation for. E.g. 'productivity', 'sales'. "
+            "Defaults to an empty string (generate the docs in the root of /pages/toolkits)"
+        ),
+        show_default=True,
+    ),
+    docs_root_dir: str = typer.Option(
+        "~/arcade/Team/docs",
```

Comment:
That's not a great default for something public - only us employees will have that.  How about no default?

Reply:
or maybe CWD?